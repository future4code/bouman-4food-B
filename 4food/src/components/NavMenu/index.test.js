import React from 'react';
import renderer from 'react-test-renderer';
import { NavMenu } from './index'

it('Snapshot NavMenu', ()=> {
    const tree = renderer
    .create(<NavMenu></NavMenu>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})