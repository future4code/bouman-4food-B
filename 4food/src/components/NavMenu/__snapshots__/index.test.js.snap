// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot NavMenu 1`] = `
<div>
  <div
    className="sc-bdVaJa fsmojC"
  >
    <span
      className="sc-bwzfXH blytJS"
    >
      Burger
    </span>
    <span
      className="sc-bwzfXH blytJS"
    >
      Asi치tica
    </span>
    <span
      className="sc-bwzfXH blytJS"
    >
      Massas
    </span>
    <span
      className="sc-bwzfXH blytJS"
    >
      Saud치veis
    </span>
    <span
      className="sc-bwzfXH blytJS"
    >
      Burger
    </span>
    <span
      className="sc-bwzfXH blytJS"
    >
      Asi치tica
    </span>
    <span
      className="sc-bwzfXH blytJS"
    >
      Massas
    </span>
    <span
      className="sc-bwzfXH blytJS"
    >
      Saud치veis
    </span>
  </div>
</div>
`;
